c2rust-analyze/src/main.rs

```diff
@@ -489,6 +489,18 @@ fn run(tcx: TyCtxt) {
                             );
                         }
                     }
+                    Rvalue::Use(Operand::Constant(c)) => {
+                        // Constants can include, for example, `""` and `b""` string literals.
+                        if let ConstantKind::Val(_, ty) = c.literal && ty.is_ref() {
+                            // The [`Constant`] is an inline value and thus local to this function,
+                            // as opposed to a global, named `const`s, for example.
+                            // This might miss local, named `const`s,
+                            acx.assign_pointer_ids(ty)
+                        } else {
+                            // TODO: Handle global, named `const`s.
+                            continue;
```

Comment:
should we emit a warning here?

Reply:
Yeah maybe.  I think this `continue` eventually turns into the same `label_no_pointers` error, which isn't very descriptive.  We should only warn on references, though, as non-references will be fine in the `label_no_pointers` later on.

Reply:
I made it into an error here: 6a0608331dfc43a9a429f7f6f73847621fb89ee6.  It'll turn into an error eventually in `label_no_pointers` anyways, and erroring here gives a much clearer error.